examples/openstack.yaml

```diff
@@ -0,0 +1,127 @@
+admin:
+  email: "a@b.c"
+  password: "verysecure"
+  sshKey: "ssh-ed25519 AAAA..."
+openstack:
+  credentials:
+    insecure: false
+
+    cloud:
+
+  baseImage:
+
+  master:
+    # Instance size for the master node(s). Example: `m1.medium`.
+    flavorName: m1.medium
+
+    rootVolume:
+      # The size of the volume in gigabytes for the root block device of master nodes.
+      size: 30
+
+  # The target OpenStack region for the cluster.
+  region: RegionOne
+
+  # Block of IP addresses used by the cluster network.
+  # This should not overlap with any other networks, such as a private datacenter connected via Direct Connect.
```

Comment:
nit: "Direct Connect" is another AWS thing, so this comment can be removed.

Reply:
Thanks. Fixed.